google/cloud/apigee/registry/v1/registry_models.proto

```diff
@@ -197,7 +197,6 @@ message ApiSpec {
   // A new revision is committed whenever the spec contents are changed.
   // The format is an 8-character hexadecimal string.
   string revision_id = 4 [
-    (google.api.field_behavior) = IMMUTABLE,
```

Comment:
We may want to hold on this until questions about "IMMUTABLE" and https://google.aip.dev/162 are resolved.